cpu/samd21/include/periph_clock_config.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2017 Daniel Evans <photonthunder@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup         cpu_samd21
+ * @{
+ *
+ * @file
+ * @brief           CPU specific definitions for generic clock generators
+ *
+ * @author          Daniel Evans <photonthunder@gmail.com>
+ */
+
+#ifndef PERIPH_CLOCK_CONFIG_H
+#define PERIPH_CLOCK_CONFIG_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief generic clock generator 1
+ * setup 1 MHz clock on generic clock generator 1
+ */
+uint8_t setup_gen1_1MHz(void);
+
+/**
+ * @brief generic clock generator 2
+ * setup 32 kHz XOSC on generic clock generator 2
+ */
+void setup_gen2_xosc32(bool standby);
```

Comment:
the standby parameter is not documented (static tests will complain about that...)